src/main/kotlin/racing/ui/RacingOutput.kt

```diff
@@ -0,0 +1,25 @@
+package racing.ui
+
+import racing.domain.RacingHistory
+
+object RacingOutput {
+    private const val SKID_MARK = "-"
+    private const val SEPARATOR = ""
+
+    fun printRacingHistories(histories: List<RacingHistory>) {
+        val stringBuilder = StringBuilder()
+
+        histories.forEach { appendSkidMarkByRound(it, stringBuilder) }
+
+        println(stringBuilder.toString())
+    }
+
+    private fun appendSkidMarkByRound(history: RacingHistory, stringBuilder: StringBuilder) {
+
+        history.records.forEach {
+            stringBuilder.append(
```

Comment:
코틀린에서 제공하는 `buildString`과 `확장 함수`를 사용한다면 다음과 같이 표현할 수 있습니다!
```suggestion
    fun printRacingHistories(histories: List<RacingHistory>) {
    
        val result = buildString {
            histories.forEach { appendSkidMarkByRound(it) }
        }

        println(result)
    }

    private fun StringBuilder.appendSkidMarkByRound(history: RacingHistory) {
        history.records.forEach {
            append(
```